README.md

```diff
@@ -81,34 +81,36 @@ Full list of command line flags (`fortio help`):
 <pre>
 Φορτίο 1.1.0 usage:
 	fortio command [flags] target
-where command is one of: load (load testing), server (starts grpc ping and http
-echo/ui/redirect/proxy servers), grpcping (grpc client), report (report only UI
-server), redirect (redirect only server), or curl (single URL debug).  where
-target is a url (http load tests) or host:port (grpc health test).  flags are:
-  -H value
+where command is one of: load (load testing), server (starts grpc ping and
+http echo/ui/redirect/proxy servers), grpcping (grpc client), report (report
+only UI server), redirect (redirect only server), or curl (single URL debug).
+where target is a url (http load tests) or host:port (grpc health test).
+flags are: .H value
```

Comment:
interesting, this is wrong but looks like the script works only on my mac and not in a docker image for instance

it's because mac is bsd fmt:
https://www.freebsd.org/cgi/man.cgi?query=fmt
which doesn't format lines starting with a .
vs
http://linuxcommand.org/lc3_man_pages/fmt1.html